frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/api/TopicResource.java

```diff
@@ -60,7 +60,15 @@ public Response createTopic(
         try {
             int partCount = Integer.parseInt(partitionCount);
             int replFactor = Integer.parseInt(replicationFactor);
-            return ResponseUtils.jsonOkResponse(cmdExecutor.createTopic(name, partCount, replFactor));
+            JSONObject result = cmdExecutor.createTopic(name, partCount, replFactor);
+            String message = result.getString("message");
+            Response.Status status;
+            if (message.contains("ERROR")) {
```

Comment:
@vibhujain check this https://github.com/apache/kafka/blob/3cdc78e6bb1f83973a14ce1550fe3874f7348b05/core/src/main/scala/kafka/admin/TopicCommand.scala#L94 
where you can see what is the message printed when a topic is created successfully

Reply:
Thanks for the suggestions. I have made appropriate changes.